osbuild/util/rhsm.py

```diff
@@ -5,18 +5,59 @@
 """
 
 import configparser
+import glob
+import os
 import re
 
 
 class Subscriptions:
     def __init__(self, repositories):
         self.repositories = repositories
+        self.fallback_method_available = False
+        self.fallback_secrets = None
+
+    @staticmethod
+    def _get_fallback_rhsm_secrets():
```

Comment:
I wonder if this should not just be a normal method and set the `fallback_secrets` if it finds them.